### ISSUE TITLE
VM deploy is failing due to libvirt 6.0 in rhel 8.2

### ISSUE THREAD

**Issue by gitsridhar** [ocp-power-automation/ocp4-upi-kvm#68]
Libvirt 6.0 was released in RHEL 8.3 last week.   Now when you do a RHEL 8.2 install, you get libvirt 6.0.   The default behavior of libvirtd has changed to use systemd socket activation.   The --listen argument is ignored, so if you want the old behavior, you need to modify the host configuration.  I have done that with this pull request for the ocs-upi-kvm project -- 

   https://github.com/ocp-power-automation/ocs-upi-kvm/pull/15

but it fails in ocp4-upi-kvm.   The PR has the ocp4-upi-kvm error messages.  Pradipta & Yussuf, have ya'll started working on libvirt 6.0?

This is blocking OCS CI builds which are dependent on ocp4-upi-kvm via ocs-upi-kvm.

**Comment by yussufsh** [comment]
As I see it from TF as a client perspective this is not an issue with the code with libvirt setup. Perhaps with an doc update to setup libvirt correctly.

BTW I have tried updating the libvirt on existing KVM box and I do not see an issue when the tls socket is masked.
```
[root@osp123 ~]# virsh version
Compiled against library: libvirt 6.0.0
Using library: libvirt 6.0.0
Using API: QEMU 6.0.0
Running hypervisor: QEMU 2.12.0
```

**Comment by gitsridhar** [comment]
@yussufsh are you going to update doc to setup libvirt correctly?

**Comment by lukebrowning** [comment]
I updated kernel, qemu, libvirt, and nftables.  I am using nftables as iptables has been deprecated for a while.  One of oddities that I worked around was that libvirtd would remove my nftables update for the new cluster.   My algorithm removes a pre-existing cluster before attempting to create a new one.   Libvirt continues deleting libvirt iptable entries that it believes are not needed.  By the time the cluster create operation gets to adding the network for the new cluster in the terraform module, all of the nftable entries for both 192.xxx networks are removed and cluster create fails.   My workaround is to restart libvirtd and add a nft rule for the new cluster between the destroy and create cluster operations.  
@yussufsh did you try to destroy a cluster and then immediately create a new one?
